sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Properties/AssemblyInfo.cs

```diff
@@ -6,8 +6,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure Data Factory Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Factory Resources.")]
-[assembly: AssemblyVersion("4.3.0.0")]
-[assembly: AssemblyFileVersion("4.4.0.0")]
+[assembly: AssemblyVersion("4.5.0.0")]
```

Comment:
`AssemblyVersion` only needs to be updated when it's a major version change.
The version in csproj needs to be updated as well.
Thanks

Reply:
@isra-fel , updated.  Thanks!